site/src/docs/components/cookie-consent/api.mdx

```diff
@@ -0,0 +1,135 @@
+[Table 5:Component object callbacks]|
+
+##### Strings and language
+The following strings can be overridden if needed. Note that you also need to provide translations if you override these. It is recommended to use the provided strings.
+
+| Property                              | Description                                                                      |
+| ------------------------------------- | -------------------------------------------------------------------------------- |
+| `texts.sections.main.title`           | Top level heading.                                                               |
+| `texts.sections.main.text`            | Main description. **The most common override because this is service-specific.** |
+| `texts.sections.details.title`        | Heading shown in the details. Seen when settings are opened.                     |
+| `texts.sections.details.text`         | Description shown in the details. Seen when settings are opened.                 |
+[Table 6:Main text content overrides]|
+
+| Property                                | Description                                                                                                           |
+| --------------------------------------- | --------------------------------------------------------------------------------------------------------------------- |
+| `ui.showSettings`                       | Text on the show/hide settings -button when settings are hidden.                                                      |
+| `ui.hideSettings`                       | Text on the show/hide settings -button when settings are shown.                                                       |
+| `ui.approveAllConsents`                 | Button text for giving consent to all cookies. Shown when settings are closed and the user cannot give optional consents. |
+| `ui.approveRequiredAndSelectedConsents` | Button text for giving consent to required cookies and selected cookies. Shown when settings are open.                |
+| `ui.approveOnlyRequiredConsents`        | Button text for giving consent to only to required cookies. All optional cookie consents are revoked.                |
+| `ui.readMore`                           | Text shown on a button when modal loses focus and is shrunk.                                                          |
+[Table 7:User interface string overrides]|
+
+| Property                    | Description                                                 |
+| --------------------------- | ----------------------------------------------------------- |
+| `tableHeadings.name`        | Title of the “hostname” column in the cookie description.  |
+| `tableHeadings.hostName`    | Title of the “path” column in the cookie description.       |
```

Comment:
`path` -> `hostName`. Interestingly, there is no path column described. Was it so that we decided to not include it or?

Reply:
fixed